doc/dev-guide.rst

```diff
@@ -163,3 +162,63 @@ root :mod:`katgpucbf` directory.
 
     The above command will likely generate a :file:`modules.rst` file, which is
     not necessary to commit.
+
+Raising the Pull Request
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For contributions from within the organisation, the steps are:
+
+1. Ensure you have the latest ``main`` in your local directory.
+2. Develop your contribution on a branch off ``main``.
+
+   a. Please prefix your branch name with the Jira ticket number. e.g. For
+      ticket NGC-1234, a suitable branch name would be
+      ``NGC-1234-task-description``.
+   b. Once you have created your branch, push it to this repository.
+
+3. Once your changes are ready for review, create the `Pull Request`_. Discuss
+   with the team who will do the review and assign it to them. If in doubt,
+   assign it to the team lead.
+
+   a. If GitHub's Pull Request page indicates there are merge conflicts ahead
+      (*Can't automatically merge*), please resolve any merge conflicts before
+      requesting a review from a developer.
+   b. A template is provided for the Pull Request description. Please fill in
+      the template with as much detail as possible and complete the checklist
+      items as indicated. You may prefix a checklist item with `(n/a)` if an
+      item is not applicable to your PR.
+
+4. This repo is monitored by the organisation's in-house Jenkins. It does an
+   end-to-end build when a Pull Request is created. Please address any issues
+   or failures reported by Jenkins and update your Reviewers accordingly.
+5. Once you have received a review, respond to any comments and apply any
+   requested changes/fixes.
+
+   a. Your intended reviewer should move the Jira ticket associated with this
+      PR back to *In Progress* and assign it back to you.
+
+6. Re-request a review of the Reviewers you've assigned to the PR and repeat
+   steps 4 and 5 until there is conclusion on Approval.
+7. Once you have received approval to merge, click **Merge pull request** with
+   its default setting of "Create a merge commit". Please delete your branch
+   after merging.
+
+For contributors outside the organisation, the steps are:
+
+1. `Fork the repository`_ on GitHub to your own account.
+2. When you deem your changes ready for review, please ensure you run the unit
+   tests locally to ensure they pass, as our Jenkins does not run on forks.
+3. Create a `Pull Request`_ from your fork to the main repository - similar to
+   the internal process described above, but without the Jira ticket steps.
+4. Address any review comments as described in step 5 above.
+5. Once you have received approval to merge, please notify a team member to
+   perform the merge on your behalf.
```

Comment:
Delete this step - if the team member approves it, they should merge it too - no need for an external user to turn around and ask for it to be merged.